geomstats/geometry/beta_distributions.py

```diff
@@ -292,6 +292,13 @@ def exp(self, tangent_vec, base_point, n_steps=N_STEPS):
         base_point = gs.to_ndarray(base_point, to_ndim=2)
         tangent_vec = gs.to_ndarray(tangent_vec, to_ndim=2)
 
+        n_base_points = base_point.shape[0]
+        n_tangent_vecs = tangent_vec.shape[0]
+        if n_base_points < n_tangent_vecs:
```

Comment:
Maybe the condition should be more strict: if n_base_points and n_tangent_vecs differ, one of them should equal 1, otherwise we may run into trouble so I think it should raise an error.

Reply:
I agree.